modules/test/playwright/tests/export-import-service/main/remoteStaging.spec.ts

```diff
@@ -135,3 +139,101 @@ test(
 		).toBeVisible();
 	}
 );
+
+test(
+	'Can publish vocabulary deletion from the Global Site using remote staging',
+	{tag: ['@LPS-89981', '@LPS-88298']},
+	async (
+		{
+			apiHelpers,
+			configStagingPage,
+			page,
+			portletStagingPage,
+			remoteApiHelpers,
+			remotePage,
+		},
+		testInfo
+	) => {
+		try {
```

Comment:
Should we move the try to the line after `await apiHelpers.jsonWebServicesStaging.enableRemoteStaging`?
Just in case it fails before we enable remote staging, we wouldn't need to tear it down